packages/arb-util/core/logreader.go

```diff
@@ -116,20 +149,24 @@ func (lr *LogReader) getLogs(ctx context.Context) error {
 
 			// Reorg happened since previous call to GetLogs.  Post-retrieve reorg of logscursor will only include
 			// extra deleted logs, won't add any new logs
-			newDeletedLogs, err := lr.cursor.LogsCursorGetDeletedLogs(lr.cursorIndex)
+			firstNewDeletedIndex, newDeletedLogs, err := lr.cursor.LogsCursorGetDeletedLogs(lr.cursorIndex)
 			if err != nil {
 				return err
 			}
 			if newDeletedLogs == nil {
 				return errors.New("missing expected deleted logs")
 			}
 
-			// Got deleted logs successfully0w
+			// Got deleted logs successfully
 			if len(newDeletedLogs) > 0 {
 				err = lr.consumer.DeleteLogs(newDeletedLogs)
 				if err != nil {
 					return err
 				}
+			} else {
+				// Logs not provided to delete
+				// TODO
+				_ = firstNewDeletedIndex
```

Comment:
I'm not sure what this todo is for. It doesn't seem like there's anything we can do in this case

Reply:
removed